diff -Naur HandBrake-b2d9b7cae68cdccdcd4f33c11bdeda98fbf37e18.old/gtk/module.rules HandBrake-b2d9b7cae68cdccdcd4f33c11bdeda98fbf37e18/gtk/module.rules
--- HandBrake-b2d9b7cae68cdccdcd4f33c11bdeda98fbf37e18.old/gtk/module.rules	2019-09-13 07:13:50.529909913 +0200
+++ HandBrake-b2d9b7cae68cdccdcd4f33c11bdeda98fbf37e18/gtk/module.rules	2019-09-13 07:16:33.099888308 +0200
@@ -22,6 +22,7 @@
 		CFLAGS="$(call fn.ARGS,GTK.GCC,.g .O *D *W ?extra)" \
 		LDFLAGS="$(call fn.ARGS,GTK.GCC,?strip .g .O ?extra.exe)" \
 		PYTHON="$(PYTHON.exe)" \
+		STRIP="$(STRIP.exe)" \
 		--prefix=$(PREFIX) \
 		--with-hb=$(call fn.ABSOLUTE,$(BUILD/))
 	$(TOUCH.exe) $@
