diff -Naur HandBrake-1.8.2.old/gtk/meson.build HandBrake-1.8.2/gtk/meson.build
--- HandBrake-1.8.2.old/gtk/meson.build	2024-10-11 19:16:00.810532466 +0200
+++ HandBrake-1.8.2/gtk/meson.build	2024-10-11 19:16:04.986563740 +0200
@@ -62,6 +62,8 @@
 endif
 
 if get_option('qsv').enabled()
+  ghb_deps += dependency('libva')
+  ghb_deps += dependency('libva-drm')
   ghb_deps += dependency('vpl')
 endif
 
diff -Naur HandBrake-1.8.2.old/libhb/rpu.c HandBrake-1.8.2/libhb/rpu.c
--- HandBrake-1.8.2.old/libhb/rpu.c	2024-10-11 19:16:00.817532518 +0200
+++ HandBrake-1.8.2/libhb/rpu.c	2024-10-11 19:16:04.984563725 +0200
@@ -11,7 +11,7 @@
 #include "handbrake/rpu.h"
 
 #if HB_PROJECT_FEATURE_LIBDOVI
-#include "libdovi/rpu_parser.h"
+#include <libdovi/rpu_parser.h>
 #endif
 
 struct hb_filter_private_s
diff -Naur HandBrake-1.8.2.old/make/configure.py HandBrake-1.8.2/make/configure.py
--- HandBrake-1.8.2.old/make/configure.py	2024-10-11 19:16:00.840532691 +0200
+++ HandBrake-1.8.2/make/configure.py	2024-10-11 19:16:04.985563732 +0200
@@ -1684,8 +1684,8 @@
         lipo       = ToolProbe( 'LIPO.exe',       'lipo',       'lipo', abort=False )
         pkgconfig  = ToolProbe( 'PKGCONFIG.exe',  'pkgconfig',  'pkg-config', abort=True, minversion=[0,27,0] )
         meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=True, minversion=[0,51,0] )
-        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=True, minversion=[2,13,0] )
-        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=True )
+        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=False, minversion=[2,13,0] )
+        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=False )
         cargo      = ToolProbe( 'CARGO.exe',      'cargo',        'cargo', abort=False )
         cargoc     = ToolProbe( 'CARGO-C.exe',    'cargo-cbuild', 'cargo-cbuild', abort=False )
 
@@ -1786,7 +1786,7 @@
     ## Additional library and tool checks
     #####################################
     # Requires oneVPL which requires CMake 3.16.3 or later
-    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=True, minversion=[3,16,3])
+    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=False, minversion=[3,16,3])
     Tools.cmake.__init__( Tools.cmake.var, Tools.cmake.option, Tools.cmake.name, **Tools.cmake.kwargs )
     Tools.cmake.run()
     for action in Action.actions:
diff -Naur HandBrake-1.8.2.old/test/module.defs HandBrake-1.8.2/test/module.defs
--- HandBrake-1.8.2.old/test/module.defs	2024-10-11 19:16:00.856532810 +0200
+++ HandBrake-1.8.2/test/module.defs	2024-10-11 19:16:14.203632765 +0200
@@ -18,10 +18,18 @@
 
 TEST.pkgconfig_libs = libass libavformat libavfilter libavcodec libavutil libswresample dvdnav \
 	dvdread libswscale theoraenc theoradec vorbis vorbisenc ogg x264 libbluray \
-	jansson libturbojpeg SvtAv1Enc
+	jansson libturbojpeg SvtAv1Enc dovi
 
 TEST.pkgconfig_libs += $(foreach m,$(MODULES.NAMES),$($m.OSL.libs))
 
+ifeq (1,$(FEATURE.qsv))
+    TEST.pkgconfig_libs += libva libva-drm vpl
+endif
+
+ifeq (1,$(FEATURE.x265))
+    TEST.pkgconfig_libs += x265
+endif
+
 ifeq (1,$(FEATURE.flatpak))
     TEST.pkgconfig_libs += glib-2.0
 endif
