diff -Naur HandBrake-1.1.0.old/libhb/muxavformat.c HandBrake-1.1.0/libhb/muxavformat.c
--- HandBrake-1.1.0.old/libhb/muxavformat.c	2018-05-17 11:16:05.894961221 +0200
+++ HandBrake-1.1.0/libhb/muxavformat.c	2018-05-17 11:17:49.026864111 +0200
@@ -800,7 +800,7 @@
                 }
                 else
                 {
-                    track->st->codecpar->codec_id = AV_CODEC_ID_SSA;
+                    track->st->codecpar->codec_id = AV_CODEC_ID_ASS;
                     need_fonts = 1;
 
                     if (subtitle->extradata_size)
@@ -1313,7 +1313,7 @@
                     free(styleatom);
                 }
             }
-            if (track->st->codecpar->codec_id == AV_CODEC_ID_SSA &&
+            if (track->st->codecpar->codec_id == AV_CODEC_ID_ASS &&
                 job->mux == HB_MUX_AV_MKV)
             {
                 // avformat requires the this additional information
diff -Naur HandBrake-1.1.0.old/libhb/stream.c HandBrake-1.1.0/libhb/stream.c
--- HandBrake-1.1.0.old/libhb/stream.c	2018-05-17 11:16:05.896961258 +0200
+++ HandBrake-1.1.0/libhb/stream.c	2018-05-17 11:17:45.139792390 +0200
@@ -5315,7 +5315,7 @@
                         "subtitle colors likely to be wrong" );
             break;
         case AV_CODEC_ID_TEXT:
-        case AV_CODEC_ID_SRT:
+        case AV_CODEC_ID_SUBRIP:
             subtitle->format = TEXTSUB;
             subtitle->source = UTF8SUB;
             subtitle->config.dest = PASSTHRUSUB;
@@ -5327,7 +5327,7 @@
             subtitle->config.dest = PASSTHRUSUB;
             subtitle->codec = WORK_DECTX3GSUB;
             break;
-        case AV_CODEC_ID_SSA:
+        case AV_CODEC_ID_ASS:
             subtitle->format = TEXTSUB;
             subtitle->source = SSASUB;
             subtitle->config.dest = PASSTHRUSUB;
@@ -5852,7 +5852,7 @@
      * either field. This is not a problem because the VOB decoder can extract this
      * information from the packet payload itself.
      *
-     * SSA subtitles (AV_CODEC_ID_SSA) do not have their duration stored in
+     * SSA subtitles (AV_CODEC_ID_ASS) do not have their duration stored in
      * either field. This is not a problem because the SSA decoder can extract this
      * information from the packet payload itself.
      */
@@ -5888,7 +5888,7 @@
             break;
     }
     if ( ffmpeg_pkt_codec == AV_CODEC_ID_TEXT ||
-         ffmpeg_pkt_codec == AV_CODEC_ID_SRT  ||
+         ffmpeg_pkt_codec == AV_CODEC_ID_SUBRIP  ||
          ffmpeg_pkt_codec == AV_CODEC_ID_MOV_TEXT ) {
         int64_t ffmpeg_pkt_duration = stream->ffmpeg_pkt.duration;
         int64_t buf_duration = av_to_hb_pts( ffmpeg_pkt_duration, tsconv, 0 );
