diff -Naur HandBrake-677b88ebbf3cb736c9f02fc59e636e4944b084bd.old/gtk/module.rules HandBrake-677b88ebbf3cb736c9f02fc59e636e4944b084bd/gtk/module.rules
--- HandBrake-677b88ebbf3cb736c9f02fc59e636e4944b084bd.old/gtk/module.rules	2018-12-08 09:18:47.414655246 +0100
+++ HandBrake-677b88ebbf3cb736c9f02fc59e636e4944b084bd/gtk/module.rules	2018-12-08 09:19:24.881320996 +0100
@@ -22,6 +22,7 @@
 		CFLAGS="$(call fn.ARGS,GTK.GCC,.g .O *D ?extra)" \
 		LDFLAGS="$(call fn.ARGS,GTK.GCC,?strip .g .O ?extra.exe)" \
 		PYTHON="$(PYTHON.exe)" \
+		STRIP="$(STRIP.exe)" \
 		--prefix=$(PREFIX) \
 		--with-hb=$(call fn.ABSOLUTE,$(BUILD/))
 	$(TOUCH.exe) $@
