diff -Naur HandBrake-9fd048196eb423965496c14dac683e6e2546e14f.old/libhb/internal.h HandBrake-9fd048196eb423965496c14dac683e6e2546e14f/libhb/internal.h
--- HandBrake-9fd048196eb423965496c14dac683e6e2546e14f.old/libhb/internal.h	2017-08-22 16:40:31.673581142 +0200
+++ HandBrake-9fd048196eb423965496c14dac683e6e2546e14f/libhb/internal.h	2017-08-22 16:40:46.382856068 +0200
@@ -8,7 +8,7 @@
  */
 
 #include "hbffmpeg.h"
-#include "extras/cl.h"
+#include <CL/cl.h>
 #ifdef USE_QSV
 #include "qsv_libav.h"
 #endif
