diff -Naur HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/libhb/rpu.c HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/libhb/rpu.c
--- HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/libhb/rpu.c	2023-09-07 13:51:21.618974386 +0200
+++ HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/libhb/rpu.c	2023-09-07 14:00:31.900235057 +0200
@@ -10,7 +10,7 @@
 #include "handbrake/handbrake.h"
 
 #if HB_PROJECT_FEATURE_LIBDOVI
-#include "libdovi/rpu_parser.h"
+#include <libdovi/rpu_parser.h>
 #endif
 
 #define RPU_DEFAULT_MODE 1
diff -Naur HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/make/configure.py HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/make/configure.py
--- HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/make/configure.py	2023-09-07 13:51:21.716976213 +0200
+++ HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/make/configure.py	2023-09-07 14:00:31.935235712 +0200
@@ -1689,9 +1689,9 @@
         libtool    = ToolProbe( 'LIBTOOL.exe',    'libtool',    'libtool', abort=True )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo',       'lipo', abort=False )
         pkgconfig  = ToolProbe( 'PKGCONFIG.exe',  'pkgconfig',  'pkg-config', abort=True, minversion=[0,27,0] )
-        meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=True, minversion=[0,49,0] )
-        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=True, minversion=[2,13,0] )
-        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=True )
+        meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=False, minversion=[0,49,0] )
+        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=False, minversion=[2,13,0] )
+        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=False )
         cargo      = ToolProbe( 'CARGO.exe',      'cargo',        'cargo', abort=False )
         cargoc     = ToolProbe( 'CARGO-C.exe',    'cargo-cbuild', 'cargo-cbuild', abort=False )
 
@@ -1792,7 +1792,7 @@
     ## Additional library and tool checks
     #####################################
     # Requires oneVPL which requires CMake 3.16.3 or later
-    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=True, minversion=[3,16,3])
+    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=False, minversion=[3,16,3])
     Tools.cmake.__init__( Tools.cmake.var, Tools.cmake.option, Tools.cmake.name, **Tools.cmake.kwargs )
     Tools.cmake.run()
     for action in Action.actions:
diff -Naur HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/test/module.defs HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/test/module.defs
--- HandBrake-c9fc5c3636646df92749754015e5afb0678bc540.old/test/module.defs	2023-09-07 13:51:21.724976362 +0200
+++ HandBrake-c9fc5c3636646df92749754015e5afb0678bc540/test/module.defs	2023-09-07 14:01:40.622519321 +0200
@@ -23,6 +23,14 @@
     TEST.GCC.l += fontconfig
 endif
 
+ifeq (1,$(FEATURE.libdovi))
+    TEST.GCC.l += dovi
+endif
+
+ifeq (1,$(FEATURE.x265))
+    TEST.GCC.l += x265
+endif
+
 ifeq (1,$(FEATURE.qsv))
     TEST.GCC.l += vpl
 ifneq (,$(filter $(HOST.system),linux freebsd))
