diff -Naur HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/libhb/rpu.c HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/libhb/rpu.c
--- HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/libhb/rpu.c	2023-10-10 07:46:42.273913958 +0200
+++ HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/libhb/rpu.c	2023-10-10 07:46:52.175091628 +0200
@@ -10,7 +10,7 @@
 #include "handbrake/handbrake.h"
 
 #if HB_PROJECT_FEATURE_LIBDOVI
-#include "libdovi/rpu_parser.h"
+#include <libdovi/rpu_parser.h>
 #endif
 
 #define RPU_DEFAULT_MODE 1
diff -Naur HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/make/configure.py HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/make/configure.py
--- HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/make/configure.py	2023-10-10 07:46:42.417916542 +0200
+++ HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/make/configure.py	2023-10-10 07:47:30.151773098 +0200
@@ -1689,9 +1689,9 @@
         libtool    = ToolProbe( 'LIBTOOL.exe',    'libtool',    'libtool', abort=True )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo',       'lipo', abort=False )
         pkgconfig  = ToolProbe( 'PKGCONFIG.exe',  'pkgconfig',  'pkg-config', abort=True, minversion=[0,27,0] )
-        meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=True, minversion=[0,51,0] )
-        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=True, minversion=[2,13,0] )
-        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=True )
+        meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=False, minversion=[0,51,0] )
+        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=False, minversion=[2,13,0] )
+        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=False )
         cargo      = ToolProbe( 'CARGO.exe',      'cargo',        'cargo', abort=False )
         cargoc     = ToolProbe( 'CARGO-C.exe',    'cargo-cbuild', 'cargo-cbuild', abort=False )
 
@@ -1792,7 +1792,7 @@
     ## Additional library and tool checks
     #####################################
     # Requires oneVPL which requires CMake 3.16.3 or later
-    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=True, minversion=[3,16,3])
+    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=False, minversion=[3,16,3])
     Tools.cmake.__init__( Tools.cmake.var, Tools.cmake.option, Tools.cmake.name, **Tools.cmake.kwargs )
     Tools.cmake.run()
     for action in Action.actions:
diff -Naur HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/test/module.defs HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/test/module.defs
--- HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf.old/test/module.defs	2023-10-10 07:46:42.427916721 +0200
+++ HandBrake-2e91369bae27841e0ffdcbe2e0fac2aaa7e779cf/test/module.defs	2023-10-10 07:46:52.179091700 +0200
@@ -23,6 +23,14 @@
     TEST.GCC.l += fontconfig
 endif
 
+ifeq (1,$(FEATURE.libdovi))
+    TEST.GCC.l += dovi
+endif
+
+ifeq (1,$(FEATURE.x265))
+    TEST.GCC.l += x265
+endif
+
 ifeq (1,$(FEATURE.qsv))
     TEST.GCC.l += vpl
 ifneq (,$(filter $(HOST.system),linux freebsd))
