diff -Naur HandBrake-1.7.2.old/libhb/rpu.c HandBrake-1.7.2/libhb/rpu.c
--- HandBrake-1.7.2.old/libhb/rpu.c	2024-01-16 13:19:01.676331256 +0100
+++ HandBrake-1.7.2/libhb/rpu.c	2024-01-16 13:19:15.258565106 +0100
@@ -10,7 +10,7 @@
 #include "handbrake/handbrake.h"
 
 #if HB_PROJECT_FEATURE_LIBDOVI
-#include "libdovi/rpu_parser.h"
+#include <libdovi/rpu_parser.h>
 #endif
 
 #define RPU_DEFAULT_MODE 1
diff -Naur HandBrake-1.7.2.old/make/configure.py HandBrake-1.7.2/make/configure.py
--- HandBrake-1.7.2.old/make/configure.py	2024-01-16 13:19:01.763332754 +0100
+++ HandBrake-1.7.2/make/configure.py	2024-01-16 13:19:47.804125462 +0100
@@ -1689,8 +1689,8 @@
         lipo       = ToolProbe( 'LIPO.exe',       'lipo',       'lipo', abort=False )
         pkgconfig  = ToolProbe( 'PKGCONFIG.exe',  'pkgconfig',  'pkg-config', abort=True, minversion=[0,27,0] )
         meson      = ToolProbe( 'MESON.exe',      'meson',      'meson', abort=True, minversion=[0,51,0] )
-        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=True, minversion=[2,13,0] )
-        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=True )
+        nasm       = ToolProbe( 'NASM.exe',       'asm',        'nasm', abort=False, minversion=[2,13,0] )
+        ninja      = ToolProbe( 'NINJA.exe',      'ninja',      'ninja-build', 'ninja', abort=False )
         cargo      = ToolProbe( 'CARGO.exe',      'cargo',        'cargo', abort=False )
         cargoc     = ToolProbe( 'CARGO-C.exe',    'cargo-cbuild', 'cargo-cbuild', abort=False )
 
@@ -1791,7 +1791,7 @@
     ## Additional library and tool checks
     #####################################
     # Requires oneVPL which requires CMake 3.16.3 or later
-    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=True, minversion=[3,16,3])
+    Tools.cmake = ToolProbe('CMAKE.exe', 'cmake', 'cmake', abort=False, minversion=[3,16,3])
     Tools.cmake.__init__( Tools.cmake.var, Tools.cmake.option, Tools.cmake.name, **Tools.cmake.kwargs )
     Tools.cmake.run()
     for action in Action.actions:
diff -Naur HandBrake-1.7.2.old/test/module.defs HandBrake-1.7.2/test/module.defs
--- HandBrake-1.7.2.old/test/module.defs	2024-01-16 13:19:01.769332857 +0100
+++ HandBrake-1.7.2/test/module.defs	2024-01-16 13:19:15.294565726 +0100
@@ -23,6 +23,14 @@
     TEST.GCC.l += fontconfig
 endif
 
+ifeq (1,$(FEATURE.libdovi))
+    TEST.GCC.l += dovi
+endif
+
+ifeq (1,$(FEATURE.x265))
+    TEST.GCC.l += x265
+endif
+
 ifeq (1,$(FEATURE.qsv))
     TEST.GCC.l += vpl
 ifneq (,$(filter $(HOST.system),linux freebsd))
