diff -Naur HandBrake-d849b13e0717396b379fb3dc3b46e85cd18a6d05.old/gtk/module.rules HandBrake-d849b13e0717396b379fb3dc3b46e85cd18a6d05/gtk/module.rules
--- HandBrake-d849b13e0717396b379fb3dc3b46e85cd18a6d05.old/gtk/module.rules	2018-06-26 09:34:37.237324058 +0200
+++ HandBrake-d849b13e0717396b379fb3dc3b46e85cd18a6d05/gtk/module.rules	2018-06-26 09:34:44.330451765 +0200
@@ -18,6 +18,7 @@
 		PKG_CONFIG_PATH=$(BUILD/)contrib/lib/pkgconfig:$(PKG_CONFIG_PATH) \
 	    CFLAGS="$(call fn.ARGS,GTK.GCC,.g .O *D ?extra)" \
 	    LDFLAGS="$(call fn.ARGS,GTK.GCC,?strip .g .O ?extra.exe)" \
+	    STRIP="$(STRIP.exe)" \
 	    --prefix=$(PREFIX) \
 	    --with-hb=$(call fn.ABSOLUTE,$(BUILD/))
 	$(TOUCH.exe) $@
